frontend/packages/dev-console/integration-tests/features/operators.feature

```diff
@@ -105,3 +105,27 @@ Scenario: Uninstall the Pipeline Operator from Operator Hub page : P-013-TC01, P
 
 Scenario: Uninstall the knative serverless operator from Operator Hub page 
    Given user is at OpenShift Serverless Operator subscription page
+
+
+@regression, @smoke
+Scenario: Install Web Terminal operator from Operator Hub page
+   Given user is at Operator Hub page with the header name "OperatorHub" 
+   When user searches for "Web Terminal"
+   And user clicks on the Web Terminal Operator card
+   And user clicks install button present on the right side bar
+   And user installs the Web Terminal operator with default values
+   Then user will see a modal with title "Web Terminal"
+   And user will see a View Operator button
+
+
+@regression, @smoke
+Scenario: Create Web Terminal instance
+   Given user is at Installed Operator page with the header name "Installed Operators"
+   And user has installed Web Terminal Operator
+   When user clicks on Web Terminal operator
+   And user clicks on All Instances tab
+   And user clicks on Create New button
+   And user clicks on Workspace
+   And user clicks on Create button
+   Then user will be redirected to Workspace tab
+   And user will see "web-terminal" instance created
```

Comment:
Useful I suppose but not related to how we integrate web terminal into console. As a user of the integrated terminal, there is no expectation to create a workspace in this manner.

Reply:
@christianvogt  I didn't get it. For now, we have to create the workspace to get the icon. Will there be change in the process and we don't gonna need it? Should I remove this?

Reply:
A console user isn't expected to manually create a Workspace. They will do so through the terminal UI by clicking the icon and filling out the form.